sentry_sdk/profiler.py

```diff
@@ -308,13 +309,11 @@ def slice_profile(self, start_ns, stop_ns):
                         )
                     current_stack.append(frames[frame])
 
-                try:
-                    stack_idx = stacks.index(current_stack)
-                except ValueError:
-                    stack_idx = len(stacks)
-                    stacks.append(current_stack)
+                if current_stack not in stacks:
+                    stacks[current_stack] = len(stacks)
```

Comment:
Isn't `current_stack` a list here? This would throw a `TypeError: unhashable type: 'list'`.

Reply:
I'm now making it into a `tuple` before using it as a key.